docs/content.zh/docs/deployment/config.md

```diff
@@ -218,6 +218,11 @@ The options in this section are necessary for setups where Flink itself actively
 
 ### Mesos
 
+{{< hint warning >}}
+Apache Mesos support was deprecated in Flink 1.13 and will become subject of removal in the future (see 
+[FLINK-22352](https://issues.apache.org/jira/browse/FLINK-22352) for further details).
```

Comment:
```suggestion
Apache Mesos support was deprecated in Flink 1.13 and is subject to removal in the future (see 
[FLINK-22352](https://issues.apache.org/jira/browse/FLINK-22352) for further details).
```

Reply:
Thanks @zentol for your review. I applied your change request on all affected sources.